qemu/tests/blockdev_inc_backup_inconsistent_bitmap.py

```diff
@@ -101,9 +104,9 @@ def do_test(self):
         self.add_persistent_bitmap()
         self.generate_inc_files(filename='inc')
         self.powerdown_and_start_vm()
-        self.check_bitmap_status(status='active')
+        self.check_bitmap_status(status=True)
         self.kill_qemu_and_start_vm()
-        self.check_bitmap_status(status='inconsistent')
+        self.check_bitmap_status(status=False)
```

Comment:
@ngu-niny  Sorry for miss your needinfo before.
For old version, two check point before and after kill_qemu_and_start_vm.
before:  status = active
after:   status = inconsistent   (in this scenario,  inconsistent=True, but it was not checked)

For the new version, with this patch, the check point are:
before:   recording = True
after:   recording = False and inconsistent=True

I have some concern about the function `check_bitmap_status` when there are multiple fields need be checked.
Suggest:
```
    def check_bitmap_field(self, **args):
        bitmap = get_bitmap_by_name(self.main_vm, self._source_nodes[0],
                                    self._bitmaps[0])
        if bitmap is None:
            self.test.fail('Failed to get bitmap %s' % self._bitmaps[0])
        else:
            for key, value in args.items():
                if value != bitmap[key]:
                    test.fail(****)
```
The call can be: `check_bitmap_filed(recording=True)`, `check_bitmap_filed(recording=False,inconsistent=True )`

Reply:
@yanan-fu Updated. Thanks for the help.